packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -324,6 +324,22 @@ export default class VList {
         }
     }
 
+    /**
+     * Get the index of the List Item in the current List
+     * @param input List item to find in the root list
+     */
+    getListItemIndex(input: Node) {
+        if (this.items) {
+            for (let index = 0; index < this.items.length; index++) {
+                const child = this.items[index];
```

Comment:
Need to check if this item is a Ordered list number and it is the first leve item. Something like:
```ts
let listIndex = this.getStart(this.listRoot) - 1;
if (child.getListType() == ListType.Ordered && child.getLevel() == 1 && !child.isDummy()) {
listIndex++;
}
if (child.getNode() == input) {
 return listIndex;
}
```

See VList.getLastItemNumber for more reference